### PR TITLE
Add tips sorted by date of publication to Tips main page

### PR DESCRIPTION
**Pull request by manshreck**

### CODE REVIEW

**Comment by tituswinters** [comment]
Hmm ... feels weird to do it this way.  I guess I'm OK with it, as a test?  Maybe ask the team?

File: _posts/2018-02-15-totw-142.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+guide used to discourage use of `explicit` for multi-parameter constructors as
+it had no meaning. That’s no longer the case.
+
+In C++11, `explicit` became meaningful for copy-initialization from braced
+lists, such as when calling a function `void f(std::pair<int, int>)` with
+`f({1, 2})` or when initializing a variable `bad` with
+`std::vector<char> bad = {“hello”, “world”};`.
+
+Wait a minute! In that last example the types don’t match. That can’t compile,
+can it? `std::vector<string> good = {“hello”, “world”};` would be reasonable,
```
Comment (tituswinters) [inline _posts/2018-02-15-totw-142.md]: std::string (throughout)

File: _posts/2018-02-15-totw-88.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/88
+type: markdown
+order: "088"
+---
+
+Originally posted as TotW #88 on Jan 27, 2015
+
+*by Titus Winters (who/titus), on behalf of the C++ Style Arbiters*
```
Comment (tituswinters) [inline _posts/2018-02-15-totw-88.md]: titus@google.com

Comment (tituswinters) [inline _posts/2018-02-15-totw-88.md]: Google C++ Style Arbiters?

File: _posts/2018-02-15-totw-88.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+what is being called and how.
+
+```c++
+std::vector<string> strings{2}; // A vector of two empty strings.
+std::vector<int> ints{2};       // A vector containing only the integer 2.
+```
+
+Second: this syntax is not exactly intuitive: no other common language uses
+something like it. The language can certainly introduce new and surprising
+syntax, and there are technical reasons why it’s necessary in some cases. The
```
Comment (tituswinters) [inline _posts/2018-02-15-totw-88.md]: necessary in some cases - *especially* in generic code.

File: _posts/2018-02-15-totw-88.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+## Conclusion
+
+The tradeoffs for uniform initialization syntax are not generally worth it: our
+compilers already warn against the Most Vexing Parse (you can use brace
+initialization or add parens to resolve the issue), and the safety from
+narrowing conversions isn’t worth the readability hit for brace-initialization
+(we’ll need a different solution for narrowing conversions, eventually). The
+Style Arbiters don’t think this issue is critical enough to make a formal rule
+on, especially because there are cases (notably in generic code) where brace
+initialization may be justified. But as general guidance, we don’t believe the
```
Comment (tituswinters) [inline _posts/2018-02-15-totw-88.md]: Maybe drop the last sentence?

File: _posts/2018-02-15-totw-88.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+  direct-initialization when available, and use parentheses over curly braces
+  when resorting to direct-initialization.)
+
+Perhaps the best overall description of the issue is Herb Sutter’s
+[GotW post](http://herbsutter.com/2013/05/09/gotw-1-solution/). Although he
+shows examples that include direct initialization of `int` with braces, his
+final advice is roughly compatible with what we present here with one caveat:
+where Herb says "where you prefer to see only the = sign", we unambiguously
+prefer to see exactly that.
+
```
Comment (tituswinters) [inline _posts/2018-02-15-totw-88.md]: In conjunction with more consistent use of `explicit` on multi-parameter constructors (see TotW 142) this provides a balance between readability, explicitness, and correctness.

File: _posts/2018-02-15-totw-142.md (hunk 1 above)
Comment (manshreck) [reply _posts/2018-02-15-totw-142.md]: Done in https://github.com/abseil/abseil.github.io/pull/145

File: _posts/2018-02-15-totw-88.md (hunk 2 above)
Comment (manshreck) [reply _posts/2018-02-15-totw-88.md]: Done in https://github.com/abseil/abseil.github.io/pull/145

Comment (manshreck) [reply _posts/2018-02-15-totw-88.md]: Done in https://github.com/abseil/abseil.github.io/pull/145

File: _posts/2018-02-15-totw-88.md (hunk 3 above)
Comment (manshreck) [reply _posts/2018-02-15-totw-88.md]: Done in https://github.com/abseil/abseil.github.io/pull/145

File: _posts/2018-02-15-totw-88.md (hunk 4 above)
Comment (manshreck) [reply _posts/2018-02-15-totw-88.md]: Done in https://github.com/abseil/abseil.github.io/pull/145

File: _posts/2018-02-15-totw-88.md (hunk 5 above)
Comment (manshreck) [reply _posts/2018-02-15-totw-88.md]: Done in https://github.com/abseil/abseil.github.io/pull/145

**Comment by manshreck** [comment]
I'm reverting this PR (though did respond and resolve the issues at hand). I've since broken out the pieces into relevant branches and have created separate pull requests

**Pull request closed without merging**